src/components/animated/_glow.less

```diff
@@ -0,0 +1,95 @@
+:root {
+  --light: 80;
```

Comment:
Instead of adding these variables on root you should add them as less variables just like other variables in the files (You can refer to `src/sbuttons.less` for this

Reply:
Sure, will do that

Reply:
Should I add them in src/sbuttons.less itself?

Reply:
Yes add them there

Reply:
We will be moving all variables to a different file later on

Reply:
Okay

Reply:
Did that, can you check and confirm?